lua/ulib/server/ucl.lua

```diff
@@ -414,6 +415,7 @@ function ucl.groupAllow( name, access, revoke )
 
 		ucl.saveGroups()
 
+		hook.Call( ULib.HOOK_GROUP_ACCESS_CHANGE, _, name, access, revoke or false )
```

Comment:
I am just used to something not being there throwing errors. It is a bit of a force thing and piece of mind. I know if it throws nil, then it will become false.